actionsystemtest/lib/action_system_test/test_helpers/active_job_setup.rb

```diff
@@ -0,0 +1,25 @@
+module ActionSystemTest
+  module TestHelpers
+    # Active Job setup helper for system testing
+    module ActiveJobSetup
+      # Sets the queue adapter to async and remembers the original adapter
+      # setting.
+      #
+      # This method should be used in the test setup code so that jobs run
+      # when the test runs because the test can't wait to check for pass/fail
+      # if it depends on running jobs.
```

Comment:
I'm not sure I understand this. Are we doing this to make sure that people don't run this with a synchronous runner? Async is already the Rails default, right? I don't think we need to do this work at all then?